pages/common/keep-header.md

```diff
@@ -0,0 +1,20 @@
+# keep-header
+
+> Convenience utility that runs Unix commands in a header-aware fashion.
+> More information: <https://github.com/eBay/tsv-utils#keep-header>.
+
+- Pass all but the first line to command and keep the first line:
+
+`keep-header {{path/to/file}} -- {{command}}`
+
+- Sort files except for retaining a single header line:
```

Comment:
How is it sorting?
alphabetically, line by line?

Reply:
Well it sorts using `sort` so the general `sort` behaviour applies. Does that make sense?

Reply:
:facepalm: completely forgot about the sort command, I thought it a feature of the command.

Reply:
No worries, that's probably down to me not explaining the purpose well!